src/components/experiment-tracking/run-dataset/index.js

```diff
@@ -94,7 +95,7 @@ function buildDatasetDataMarkup(
             })}
             key={value.runId + index}
           >
-            {value.value}
+            {typeof value === 'object' ? JSON.stringify(value) : value.value}
```

Comment:
This is a temporary fix to resolve values that returns an object ( which is different from the design that the FE built with assumption on)

Reply:
Got it.

Can you please push your changes you added when we were sitting together at your desk?